react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -48,6 +48,7 @@ export default class ProofsRender extends Component {
 
     const statusColor = {
       normal: colors.lightBlue,
+      loggedOut: colors.lightBlue,
```

Comment:
I'm going to apply the style guide to this all soon

On Monday, January 11, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/tracker/proofs.render.desktop.js
> <https://github.com/keybase/client/pull/1701#discussion_r49388780>:
>
> > @@ -48,6 +48,7 @@ export default class ProofsRender extends Component {
> >
> >      const statusColor = {
> >        normal: colors.lightBlue,
> > +      loggedOut: colors.lightBlue,
>
> I just guessed here, not sure what the design actually calls for
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1701/files#r49388780>.
>